core/src/main/java/org/commonjava/maven/ext/core/impl/BaseGroovyManipulator.java

```diff
@@ -184,7 +186,15 @@ void applyGroovyScript( List<Project> projects, Project project, File groovyScri
             }
             catch ( Exception e )
             {
-                throw new ManipulationException( "Unable to parse script", e );
+                //noinspection ConstantConditions
+                if ( e instanceof ManipulationException )
+                {
+                    throw ((ManipulationException)e);
+                }
+                else
+                {
+                    throw new ManipulationException( "Unable to parse script", e );
```

Comment:
@paulgallagher75 I wonder if this exception would be better phrased as `Problem running script` ?